src/test/java/org/broadinstitute/hellbender/utils/codecs/refseq/RefSeqCodecUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.hellbender.utils.codecs.refseq;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RefSeqCodecUnitTest {
+
+    @Test
+    public void testCanDecode() {
+        final String EXTRA_CHAR = "1";
+        RefSeqCodec codec = new RefSeqCodec();
+        Assert.assertTrue(codec.canDecode("filename." + RefSeqCodec.FILE_EXT));
+        Assert.assertTrue(codec.canDecode("filename" + EXTRA_CHAR + "." + RefSeqCodec.FILE_EXT));
+        Assert.assertFalse(codec.canDecode("filename." + RefSeqCodec.FILE_EXT + EXTRA_CHAR));
+        Assert.assertFalse(codec.canDecode("filename" + RefSeqCodec.FILE_EXT));
+    }
```

Comment:
Weren't there any RefSeq tests from GATK3 that we could port over?

Reply:
Yeah here they all are: 
```
public class RefSeqCodecUnitTest {

    @Test
    public void testCanDecode() {
        final String EXTRA_CHAR = "1";
        RefSeqCodec codec = new RefSeqCodec();
        Assert.assertTrue(codec.canDecode("filename." + RefSeqCodec.FILE_EXT));
        Assert.assertTrue(codec.canDecode("filename" + EXTRA_CHAR + "." + RefSeqCodec.FILE_EXT));
        Assert.assertFalse(codec.canDecode("filename." + RefSeqCodec.FILE_EXT + EXTRA_CHAR));
        Assert.assertFalse(codec.canDecode("filename" + RefSeqCodec.FILE_EXT));
    }
}
```